kitsune/messages/jinja2/messages/outbox.html

```diff
@@ -14,8 +14,14 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
         </div>
       {% if msgs %}
         <form action="{{ url('messages.outbox_bulk_action') }}" method="POST">
+        <div class="sumo-button-wrap">
+            <button type="submit" name="delete" class="sumo-button">{{ _('Delete Selected') }}</button>
+        </div>
         <ol class="outbox-table">
             <li class="email-row header">
+                <div class="email-cell check">
+
+                </div>
```

Comment:
Nit. Eliminate whitespace.
```suggestion
                <div class="email-cell check"></div>
```